tasks.py

```diff
@@ -16,16 +16,17 @@ def requirements(ctx):
 
 @task
 def lint(ctx, fix=False):
-    ctx.run(
-        f"python -m black --target-version py36 {'--check' if not fix else ''} ./*.py ./src ./tests",
-        pty=True,
-    )
-
-    if fix:
-        print("\nSkipping type check as there is no fixer")
-    else:
-        print("\nRunning type check")
-        ctx.run("python -m mypy --strict src", pty=True)
+    skip_command = "echo 'Skipping'"
+    lint_commands = {
+        "isort": f"python -m isort {'' if fix else '--check-only --diff'} -y",
+        "black": f"python -m black {'' if fix else '--check'} .",
+        "mypy": None if fix else f"python -m mypy --strict src",
```

Comment:
We should run mypy always, otherwise you end up running the linter task twice, once to fix and once to check for mypy issues.

Reply:
Sure, I was preserving the previous behaviour in here